gemma/gemma-2-27b-it-vllm/config.yaml

```diff
@@ -3,6 +3,7 @@ python_version: py311
 model_metadata:
   example_model_input: {"prompt": "what is the meaning of life"}
   main_model: google/gemma-2-27b-it
+  repo_id: https://huggingface.co/google/gemma-2-27b-it
```

Comment:
by convention we only do `google/gemma-2-27b-it`; i wonder if we can get rid of main_model above, and just reference `repo_id`?